packages/components/CHANGELOG-FIGMA-COMPONENTS.md

```diff
@@ -1,5 +1,30 @@
 # [HDS Product - Components](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?type=design&node-id=6790-10926&mode=design&t=Ps0aMGZ6F3z7bAJ4-0)
 
+## August 2nd, 2024
+
+`ApplicationState` - multiple enhancements to the `ApplicationState` including:
+
+- Added support for a media slot, which when enabled, will be added above the title, allowing yielded illustrations to occupy that space.
+- Added an `alignment` property which can be set at the root level to `left` or `center`. This adjusts the component’s text, footer actions, and media alignment.
+- The footer now supports up to three actions at once, modeled after the Button Set. Previously the actions were justified (one all the way left and one all the way right). Supported actions now include Dropdown, Button, and Standalone Link. Previously Standalone Link was the only supported option.
```

Comment:
To clarify this maybe explain the layout of the 3 options? It says previously the layout (of 2 options) was justified but doesn't explain how/if that changed.

Reply:
Reminder: If you update text here as suggested, also update the duplicated text below.